apps/test/unit/gamelab/shapesTest.js

```diff
@@ -0,0 +1,58 @@
+import {throwIfSerializedAnimationListIsInvalid} from '@cdo/apps/gamelab/shapes';
```

Comment:
is this still a good name for this function?

Reply:
That's a good question! After some thought, I think it still makes sense. The majority of this function is throwing errors if the list is invalid. I also think it makes sense to keep this de-duplication in the function as it is part of the validation of the list. I think the purpose of the function remains unchanged - before and after this PR, if this function is called with a list and doesn't throw an error, then we know that the list is valid.

Reply:
great, thanks for being so thoughful